scripts/checkpatch.pl

```diff
@@ -6527,6 +6541,13 @@ sub process {
 			}
 		}
 
+# check for #defines of various API specifying things like _POSIX_C_SOURCE
```

Comment:
```suggestion
# check for feature test macros that request C library API extensions, violating rules A.4 and A.5
```

Reply:
always best to say 'why' and not 'what' in your commit messages. thanks for the fix.